### PR TITLE
Publish 1.24.0 documentation updates

### PR DESCRIPTION
**Pull request by keirthana**
This PR consolidates all changes from `next-release` branch to publish the documentation updates for 1.24.0.

### CODE REVIEW

File: explanation/anbox-cloud.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ See the following table for a comparison of features for the different variants:
 
 *\* When purchasing the Anbox Cloud Appliance through the AWS Marketplace, the Ubuntu Pro subscription does not include vendor support.*
 
+*\*\* Android 14 images are available only for AOSP images and not AAOS.*
+
```
Comment (ajanon) [inline explanation/anbox-cloud.md:39]: Wait, this is not accurate, we have AAOS images for A14

File: reference/provided-images.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ The following table lists supported images available on the official image serve
 
 | Name                        | Based on | Android Version | Ubuntu Version | Available since |
 |-----------------------------|----------|-----------------|----------------|---------------|
+| `jammy:android14:amd64`     | AOSP     | 14              | 22.04          | 1.24.0 |
+| `jammy:android14:arm64`     | AOSP     | 14              | 22.04          | 1.24.0 |
```
Comment (ajanon) [inline reference/provided-images.md:15]:
```suggestion
| `jammy:aaos14:amd64`     | AAOS     | 14              | 22.04          | 1.24.0 |
| `jammy:aaos14:arm64`     | AAOS     | 14              | 22.04          | 1.24.0 |
| `jammy:android14:amd64`     | AOSP     | 14              | 22.04          | 1.24.0 |
| `jammy:android14:arm64`     | AOSP     | 14              | 22.04          | 1.24.0 |
```

**Review by ajanon — CHANGES_REQUESTED** [review]

**Comment by adglkh** [comment]
LGTM, after comments from Alexis are resolved

**Review by adglkh — APPROVED** [review]
LGTM

File: reference/provided-images.md (hunk 2 above)
Comment (keirthana) [reply reference/provided-images.md:15]: Oh, sorry, the [old PR description](https://github.com/canonical/anbox-cloud-docs/pull/190) that I wrote tripped me up. I added this but then removed it looking at the PR description. I will fix it.

**Review by ajanon — APPROVED** [review]
LGTM, thanks for the quick change!

**Pull request merged**